test/demo/createToast.inttest.ts

```diff
@@ -87,3 +86,17 @@ describe('When calling createNotification with the Notification Center not showi
         await assertNotificationStored(testWindow.identity, note);
     });
 });
+
+describe('When calling createNotification with the Notification Center muted and not showing', () => {
+    beforeAll(toggleCenterMuted);
+    afterAll(toggleCenterMuted);
+
+    test('No toast is shown for the notification', async () => {
+        // The toast creation is not awaited as part of notes.create, so we add a small
+        // delay to allow the toast time to spawn.
+        await delay(Duration.TOAST_DOM_LOADED);
```

Comment:
The test in createCenter uses `Duration.TOAST_CREATE` here. Any reason for the difference?..

Reply:
Consistency with surrounding tests. Makes sense to me that they'd be different, given one is just adding some HTML, the other is spinning up a whole new window

Reply:
Gone with TOAST_DOM_LOADED for when waiting for toast window